Cooper/ClientApp/src.v2/app/components/my-profile/my-profile.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { User } from '@models';
+
+@Component({
+  selector: 'coop-my-profile',
+  templateUrl: './my-profile.component.html',
+  styleUrls: ['./my-profile.component.css']
+})
+export class MyProfileComponent implements OnInit {
+
+  editMode = false;
```

Comment:
also can be Input() with default value false

Reply:
But that variable is not using in foreign-profile.component. Is it okay to have it in layout only for one dummy component?